docs/usage.md

```diff
@@ -55,10 +85,12 @@ The Image Viewer widget allows users to display images or diagrams that are rele
 
 The primary purpose of the *Global Visibility* playbook collection is to gather data from individual sites and transform it into a format that is easily consumable by the widgets. By centralizing data within the "Global Visibility" module, the solution pack's widgets can efficiently retrieve and display information, providing users with a unified view of critical metrics and records across sites.
```

Comment:
The *Global Visibility* playbook collection gathers data from individual sites and transform it into a format that is easily consumable by the widgets. By centralizing data within the **Global Visibility** module, the solution pack's widgets can efficiently retrieve and display information, providing you with a unified view of critical metrics and records across sites.